tests/e2e/scale/test_pvc_creation_deletion_scale.py

```diff
@@ -69,192 +126,122 @@ def test_multiple_pvc_creation_deletion_scale(self, namespace, access_mode, inte
         )
 
         # Check for PVC status using threads
-        threads = list()
-        for obj in pvc_objs:
-            process = threading.Thread(
-                target=helpers.wait_for_resource_state,
-                args=(obj, constants.STATUS_BOUND, )
-            )
-            process.start()
-            threads.append(process)
-        for process in threads:
-            process.join()
-
+        helpers.handle_pvc_scale_threading(
+            self.pvc_objs,
+            (lambda x: helpers.wait_for_resource_state),
```

Comment:
are we missing helpers function? since this is an arg why not direct arg , same for other helper function args

Reply:
The code in the second section has been switched back to not do these calls.